lib/OpenQA/Utils.pm

```diff
@@ -649,6 +650,48 @@ sub find_job {
     return $job;
 }
 
+sub read_test_modules {
```

Comment:
I am not so sure if the "utils" file is really the right one for such route specific implementation. Also the file "Utils.pm" is getting a bit big. Anyone has an idea for a better location for a function used both in the webUI route as well as the API one?

Reply:
Made more sense to have it in the Utils module, pretty much like find_job does. But I agree that Utils is growing up a bit.